testdata/p4_14_samples_outputs/parser_pragma2.p4-stderr

```diff
@@ -1,6 +1,6 @@
 parser_pragma2.p4(18): [--Wwarn=parser-transition] warning: SelectCase: unreachable
         0xAB00 : Cowles;
         ^^^^^^
-parser_pragma2.p4(16): [--Wwarn=parser-transition] warning: 'ListExpression': transition does not depend on select argument
+parser_pragma2.p4(16): [--Wwarn=parser-transition] warning: '{ tmp }': transition does not depend on select argument
```

Comment:
This is one problem with this improvement: this error message no longer makes sense.

Reply:
I noticed that too and I am not sure of a good way to fix this. Is this problem unique to `ListExpression` components, or is it a problem when printing compiler-generated `Expression`s in other places, too?

Reply:
I noticed that too and I am not sure of a good way to fix this. Is this problem unique to `ListExpression` components, or is it a problem when printing compiler-generated `Expression`s in other places, too?

Reply:
No, it happens in other places as well.